### ISSUE TITLE
Incorrect spacing of default gui on Windows

### ISSUE THREAD

**Issue by ennerf** [hydraulic-software/conveyor#98]
I'm aware that it's not a supported feature, but probably good to know in case it'll become one. With 3 CLI tools the default gui on Windows 10 at 1080p resolution shows the update button out of bounds:

![image](https://github.com/hydraulic-software/conveyor/assets/5491587/f307d346-6e98-479f-b6f9-4f94478de144)

On Windows 11 a highres display (2256 x 1504 @ 150% scale) it renders within bounds, although it looks like it's mainly due to some styling differences.

![image (13)](https://github.com/hydraulic-software/conveyor/assets/5491587/1c703809-f73a-4542-b14e-c0ac68009a43)

**Setup**
- [Conveyor v13.1](https://github.com/HebiRobotics/QuickBuffers/blob/7941ddac96ebf1a23c554633c0af33a7bb736e27/.github/workflows/release.yml#L175-L191)
- [conveyor.conf](https://github.com/HebiRobotics/QuickBuffers/blob/7941ddac96ebf1a23c554633c0af33a7bb736e27/conveyor.conf)
- [binaries](**url**)

**Comment by mikehearn** [comment]
Yeah good suggestion. I guess there's the question of what the background on the left hand side should be if the window can grow vertically.

Alternatively, maybe we shouldn't be listing the commands there? There isn't much prior art when it comes to a GUI for CLI apps 🤣
